src/main/java/practice/StreamPractice.java

```diff
@@ -1,20 +1,34 @@
 package practice;
 
-import java.util.Collections;
+import java.util.Arrays;
+import java.util.Collection;
 import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 import model.Candidate;
+import model.Cat;
 import model.Person;
 
 public class StreamPractice {
+    private CandidateValidator validator = new CandidateValidator();
```

Comment:
```suggestion
    private CandidateValidator candidateValidator = new CandidateValidator();
```
it should be method local variable